frontend/src/metabase-lib/lib/Question.ts

```diff
@@ -931,17 +931,16 @@ export default class Question {
       !question.id() ||
       (originalQuestion && question.isDirtyComparedTo(originalQuestion))
     ) {
-      return Urls.question(
-        null,
-        question._serializeForUrl({
+      return Urls.question(null, {
+        hash: question._serializeForUrl({
           clean,
           includeDisplayIsLocked,
           creationType,
         }),
         query,
-      );
+      });
```

Comment:
Keeping this as is as `_serializeForUrl` method and `Urls.serializedCard` have different serialization implementation